packages/deprecation-crawler/src/lib/tasks/ensure-crawler-tsconfig.ts

```diff
@@ -0,0 +1,73 @@
+import { CrawlConfig } from "../models";
+import { prompt } from "enquirer";
+import { normalize } from "path";
+import { TSCONFIG_PATH } from "../constants";
+import { findTsConfigFiles } from "../config";
+import { readFileSync, writeFileSync } from "fs";
+import { format, resolveConfig } from "prettier";
+import { fileExists } from "@nrwl/workspace/src/utils/fileutils";
+
+export async function ensureTsConfigPath(config: CrawlConfig): Promise<CrawlConfig> {
+
+  const tsConfigFiles: string[] = findTsConfigFiles();
```

Comment:
the `findTsConfigFiles` method can be moved to this file, as its the only place where we need it.

Reply:
moved here. How did you detect that? Manually? Or over ide?

Reply:
I just knew it 😅